meinberlin/config/settings/dev.py

```diff
@@ -23,7 +23,10 @@
 except ImportError:
     pass
 else:
-    INSTALLED_APPS += ("debug_toolbar",)
+    INSTALLED_APPS += (
+        "debug_toolbar",
+        "meinberlin.apps.dev",
```

Comment:
should we move this in it's own block? so it's not tied to the import of the debug_toolbar?

Reply:
i feel, it'd look more weird to have the installed_apps there twice. Also they are both dev related tools.